workers/deployment_manager/DeploymentManager/default_launch.json

```diff
@@ -0,0 +1,60 @@
+{
+  "template": "medium_entity_db_v2",
```

Comment:
Just `medium` 😉

Reply:
that one gets copy-pasted from the root folder. via the "start-deployment-manager", so would prefer to leave it like this until I rebased the base branch.